contrib/themes/ingrid.toml

```diff
@@ -0,0 +1,44 @@
+"attribute" = "#839A53"
```

Comment:
I think we should put some comments on the contributor, so later we can check if the color theme is still in use.

Reply:
Agreed, 

```
# Author: Name <email>
```

works